src/OFL-1.1.xml

```diff
@@ -6,6 +6,11 @@
          <crossRef>http://www.opensource.org/licenses/OFL-1.1</crossRef>
       </crossRefs>
       <notes>This license was released: 26 February 2007.</notes>
+      <optional>
+         <copyrightText><p>Copyright (c) &lt;dates&gt;, &lt;Copyright Holder&gt; (&lt;URL|email&gt;),
+            <br/>with Reserved Font Name &lt;Reserved Font Name&gt;.</p></copyrightText>
+         <p>This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is copied below, and is also available with a FAQ at: http://scripts.sil.org/OFL</p>
```

Comment:
The “This license is copied below…” bit is a separate paragraph [upstream][1].

[1]: http://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web